platform/docs/docs/configuration/configurationFiles.md

```diff
@@ -184,6 +184,7 @@ if auth headers are used, a preflight request is required.
 - `useSharedArrayBuffer` (default to 'TRUE', options: 'AUTO', 'FALSE', 'TRUE', note that these are strings), for volume loading we use sharedArrayBuffer to be able to
   load the volume progressively as the data arrives (each webworker has the shared buffer and can write to it). However, there might be certain environments that do not support sharedArrayBuffer. In that case, you can set this flag to false and the viewer will use the regular arrayBuffer which might be slower for large volume loading.
 - `supportsWildcard`: (default to false), if set to true, the datasource will support wildcard matching for patient name and patient id.
+- `allowMultipleExport`: (default to false), if set to true, the user will be able to select the datasource to export the report to.
```

Comment:
Hmmm... I think I prefer something like `allowMultiSelectExport`. It might just be me, but allowMultipleExport kinda suggests exporting to mulitple places ALL AT ONCE.